src/main/java/com/zufar/onlinestore/payment/service/PaymentRetriever.java

```diff
@@ -0,0 +1,29 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.zufar.onlinestore.payment.dto.PaymentDetailsDto;
+import com.zufar.onlinestore.payment.exception.PaymentNotFoundException;
+import com.zufar.onlinestore.payment.converter.PaymentConverter;
+import com.zufar.onlinestore.payment.repository.PaymentRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Objects;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class PaymentRetriever {
```

Comment:
add java doc please
it helps to understand this class for people who have no idea about stripe

Reply:
Done